host/4/bullseye/amd64/java/java11-zulu/java11-composite.template

```diff
@@ -0,0 +1,34 @@
+ARG JAVA_VERSION=11u11
+
+FROM mcr.microsoft.com/java/jre-headless:${JAVA_VERSION}-zulu-debian10-with-tools as jre
+FROM mcr.microsoft.com/dotnet/runtime-deps:6.0.0
+ARG HOST_VERSION
+
+EXPOSE 2222 80
+
+ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
+    HOME=/home \
+    FUNCTIONS_WORKER_RUNTIME=java \
+    DOTNET_USE_POLLING_FILE_WATCHER=true \
+    HOST_VERSION=${HOST_VERSION}
+
+# Fix from https://github.com/GoogleCloudPlatform/google-cloud-dotnet-powerpack/issues/22#issuecomment-729895157
+RUN apt-get update && \
+    apt-get install -y libc-dev
+
+COPY --from=runtime-image [ "/azure-functions-host", "/azure-functions-host" ]
+COPY --from=runtime-image [ "/workers/java", "/azure-functions-host/workers/java" ]
+COPY --from=jre [ "/usr/lib/jvm/zre-hl-tools-11-azure-amd64", "/usr/lib/jvm/zre-11-azure-amd64" ]
```

Comment:
@cooperlink - please verify if the copy cmds look good - to ensure layer reuse

Reply:
Will sync with Kaibo offline. Need to move the func ext bundles.

Reply:
@CooperLink updated the images as suggested, please take a look. thank you.

Reply:
@pragnagopa @CooperLink , team please let me know if the pr looks good now. thank you. Really appreciate your help!

Reply:
Looks great to me. Thanks Kaibo